project/Version.scala

```diff
@@ -0,0 +1,31 @@
+package akka
+
+import sbt._
+import sbt.Keys._
+
+/**
+ * Generate version.conf and akka/Version.scala files based on the version setting.
+ */
+object Version {
+
+  def versionSettings: Seq[Setting[_]] = inConfig(Compile)(Seq(
+    resourceGenerators <+= generateVersion(resourceManaged, _ / "version.conf",
+      """|akka.version = "%s"
+         |"""),
+    sourceGenerators <+= generateVersion(sourceManaged, _ / "akka" / "Version.scala",
+      """|package akka
+         |
+         |object Version {
+         |  val current: String = "%s"
```

Comment:
Yes, could be a constant (with `final` and without type). Should then become an inlined string literal in `ActorSystem.Version`. Shall I make that change?